### PR TITLE
adding styling for ordered list items

### PR DESCRIPTION
**Pull request by ciaranlp**
Numbered / Ordered List Items were missing the styling that was in place for Unordered List Items

### CODE REVIEW

**Comment by brian-learningpool** [comment]
+1

File: src/core/less/reset.less [hunk 1]
```diff
@@ -8,7 +8,7 @@ outline: 0; font-size: 100%; vertical-align: baseline; background: transparent;}
 article, aside, figure, footer, header, hgroup, nav, section {display: block;}
 img, object, embed {max-width: 100%;}
 html {overflow-y: scroll;}
-ul li {margin-left:20px;
+ul li, ol li {margin-left:20px;
```
Comment (ryangadams) [inline src/core/less/reset.less]: do you need the ul and ol?

**Comment by moloko** [comment]
+1

File: src/core/less/reset.less (hunk 1 above)
Comment (ciaranlp) [inline src/core/less/reset.less]: yes, better safe than sorry.

Comment (ryangadams) [inline src/core/less/reset.less]: ok, boss. 

+1

**Comment by dennis-learningpool** [comment]
+1

**Comment by oliverfoster** [comment]
+1

**Pull request merged**